lxd/daemon_images.go

```diff
@@ -26,6 +26,7 @@ import (
 // Simplestream cache
 type imageStreamCacheEntry struct {
```

Comment:
Yea, now I know. =) Isn't it all Ubuntu stuff - can you ask why did they choose so unfortunate name? Can it be renamed to JSONOSImage or something?